spec/README.mediawiki

```diff
@@ -714,83 +714,65 @@ The client is responsible for tracking the sequence ID to ensure all order
 updates are received. If an update appears to be missing, the client should
 re-subscribe to the market to synchronize the order book from scratch.
 
-<code>result</code>
-{|
-! field       !! type   !! description
-|-
-| book  || object || An OrderBook object
-|-
-| seq   || int    || A sequence ID
-|}
-
-'''JSON OrderBook object'''
+'''Response'''
 
+<code>payload</code>
 {|
 ! field    !! type !! description
 |-
-| marketid || int || the market ID. included with order book updates.
+| seq      || int    || A sequence ID
 |-
-| epoch    || int  || the current epoch
+| marketid || string || An identifier used to identify this market in notifications
 |-
-| orders   || &#91;object&#93; || A list of Order objects
+| orders   || &#91;object&#93; || A list of '''Order''' objects (described below)
 |}
 
 '''JSON Order object'''
 
 {|
-! field   !! type   !! description
+! field   !! type   !! description !! notes
 |-
-| oid     || string || the order ID
+| seq      || int   || A sequence ID || superceded in <code>orderbook</code> response
 |-
-| otype   || string || "l" for ''limit'', "m" for ''market''
+| marketid || string || An unique market identifier ||
 |-
-| side    || string || "b" for ''buy'', "s" for ''sell''
+| oid     || string || the order ID ||
 |-
-| osize   || int    || order size (atoms)
+| otype   || string || "l" for ''limit'', "m" for ''market'', "c" for cancel || epoch_order only
 |-
-| rate    || int    || price rate (quote asset per unit base asset, atoms). only set on limit orders
+| side    || string || "b" for ''buy'', "s" for ''sell'' || epoch_order, book_order
 |-
-| tif     || string || time in force. one of "i" for ''immediate'' or "s" for ''standing''. only set on limit orders
+| osize   || int    || order size (atoms) || epoch_order, book_order
```

Comment:
I would prefer to use some form of "quantity" here.

Reply:
Want to make this `oamount` or `oquantity`, or just change the description?

Reply:
@buck54321 I may have misunderstood your comment.  Did you want to adjust the wording here?

Reply:
Rebase needed now too, thanks.

Reply:
Is `qty` obvious enough?

Reply:
Yeah, super clear.